homeassistant/components/withings/api.py

```diff
@@ -63,7 +63,7 @@ def _request(
         )
         return response.json()
 
-    async def _do_retry(self, func, attempts=3) -> Any:
+    async def _do_retry(self, func: Callable, attempts=3) -> Any:
```

Comment:
This typing is kinda meh but since its likely going away anyways it probably not worth improving and than throwing away.